### PR TITLE
Enable LZ4 support.

### PR DESCRIPTION
**Pull request by petermattis**
LZ4 is supposed to be faster than Snappy.

### CODE REVIEW

File: cgo_flags.go [hunk 1]
```diff
@@ -5,8 +5,8 @@ package rocksdb
 
 // #cgo CPPFLAGS: -Iinternal -Iinternal/include -Iinternal/db -Iinternal/util
 // #cgo CPPFLAGS: -Iinternal/utilities/merge_operators/string_append
-// #cgo CPPFLAGS: -I../c-snappy/internal
-// #cgo CPPFLAGS: -DROCKSDB_PLATFORM_POSIX -DNDEBUG -DSNAPPY
+// #cgo CPPFLAGS: -I../c-snappy/internal -I../c-lz4/internal/lib
```
Comment (bdarnell) [inline cgo_flags.go:8]: Don't we need to add c-lz4 to the glockfile first?

**Comment by spencerkimball** [comment]
LGTM

**Comment by bdarnell** [comment]
LGTM

File: cgo_flags.go (hunk 1 above)
Comment (petermattis) [inline cgo_flags.go:8]: I don't think that matters. The glockfile still points to the c-rocksdb that doesn't require lz4. Am I missing something?

Comment (bdarnell) [inline cgo_flags.go:8]: Ah, this PR is in the c-rocksdb repo. Nevermind.

**Pull request merged**